src/FormBuilder/resources/js/form-builder/src/styles/_components.scss

```diff
@@ -155,6 +155,32 @@ This creates a consistent separator between the tabs and the rest of the sidebar
     padding-right: 16px;
 }
 
+.components-panel__body .components-notice {
+    color: var(--givewp-grey-700);
+    margin-left: 0;
+    margin-right: 0;
+    padding: var(--givewp-spacing-3) var(--givewp-spacing-4);
+
+    .components-notice__content {
+        margin: 0;
+
+        * {
+            font-size: 0.75rem;
+            line-height: 1rem;
+        }
+    }
+
+    h4 {
+        color: var(--givewp-grey-900);
+        margin: 0 0 var(--givewp-spacing-2);
+    }
+
+    .components-notice__action.is-link {
+        color: var(--givewp-grey-900);
+        margin-left: 0;
+    }
+}
```

Comment:
Is this for this settings specifically?  If not it will be applied to all notice components.  If it's specific to this setting I would suggest importing a standalone stylesheet into the block component.